src/web.ts

```diff
@@ -1,22 +1,61 @@
-import { WebPlugin } from '@capacitor/core';
-import { SignInWithApplePlugin } from './definitions';
+import { WebPlugin } from "@capacitor/core";
 
-export class SignInWithAppleWeb extends WebPlugin implements SignInWithApplePlugin {
+import { SignInWithApplePlugin, InitOptions } from "./definitions";
+
+declare var window: any;
+
+export class SignInWithAppleWeb extends WebPlugin
+  implements SignInWithApplePlugin {
   constructor() {
     super({
-      name: 'SignInWithApple',
-      platforms: ['web']
+      name: "SignInWithApple",
+      platforms: ["web"],
     });
   }
 
-  async Authorize(): Promise<{response: any}> {
+  async SignIn(): Promise<void> {
+    if (window && window.AppleID) {
+      return await window.AppleID.auth.signIn();
+    }
+
     return;
   }
+
+  async Init(options: InitOptions): Promise<void> {
```

Comment:
Guard against reloading the script multiple times, use an `initialised` flag (or just check the presense of `window.AppleID` first)

Reply:
Addressed in c58f550